python/src/opendp/context.py

```diff
@@ -210,8 +249,19 @@ def unit_of(
     U=None,
 ) -> Tuple[Metric, float]:
     """Constructs a unit of privacy, consisting of a metric and a dataset distance. 
-
-    :param ordered: Set to true to use InsertDeleteDistance instead of SymmetricDistance, or HammingDistance instead of ChangeOneDistance.
+    The parameters are mutually exclusive.
+
+    >>> unit_of(contributions=3)
+    (SymmetricDistance(), 3)
+    >>> unit_of(l1=2.0)
+    (L1Distance(f64), 2.0)
+
+    :param contributions: TODO
```

Comment:
TODOs: work with @Shoeboxam to fill in before merging

Reply:
Thanks for descriptions. Done.